src/Pages/Quote.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import NavBar from '../component/Navbar';
+
+const Quote = () => (
+  <div>
+    <NavBar />
```

Comment:
- [ ] kindly remove the `navbar` in this component and import it in the app.js to call it for once instead of calling it in each component